### ISSUE TITLE
Exponential component not working and missing arguments

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#1660]
First off, shift tab in `hs.model.components1D.Exponential()` does not provide arguments.

Secondly, when trying to fit it, it returns errors of divide by zero and overflow.
It was failing on my dataset as well, but here is an example using a hs example signal where I would expect it to fit something.:
```
m = hs.datasets.example_signals.EDS_SEM_Spectrum().create_model(auto_background=False)
exp = hs.model.components1D.Exponential()
m.append(exp)
m.fit()
m.plot()
```
Creating my own Expression works fine.

**Comment by francisco-dlp** [comment]
I think that the component works fine (except for the lack of constructor arguments), the issue is that the default value of the parameters is 0.

**Comment by ericpre** [comment]
The arguments issue have been fixed in #2135 and the component works fine indeed.
